src/main/java/org/example/ValidParentheses.java

```diff
@@ -0,0 +1,40 @@
+package org.example;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Stack;
+
+public class ValidParentheses {
+    private static final HashMap<Character,Character> bracketsMap = new HashMap<>();
```

Comment:
It is better to use abstract data type when declaring a variable
```suggestion
    private static final Map<Character,Character> bracketsMap = new HashMap<>();
```